src/Concordium/Client/Types/TransactionStatus.hs

```diff
@@ -25,6 +27,31 @@ data TransactionStatusResult' a = TransactionStatusResult
 
 type TransactionStatusResult = TransactionStatusResult' ValidResult
 
+-- |Convert an `TransactionStatus` instance into a `TransactionStatusResult` instance.
```

Comment:
```suggestion
-- |Convert a `TransactionStatus` instance into a `TransactionStatusResult` instance.
```

Reply:
Also, this is not Markdown. Use the correct Haddock markup please.

Reply:
I will have to go through all the comments to ensure they are proper Haddock markup. I added a separate TODO for this, and I will do this for the entire feature branch.